src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -68,8 +71,26 @@ class FeelingRoom extends Component {
   }
   createSadRoom = () => {
     //functions to animate go here
+    console.log('sad')
```

Comment:
I've seen a few stray console.logs in the last few pull requests, let's make sure to do a sweep sometime soon.

Reply:
Yes!
Added that to the wednesday to do list!